--- TeXmacs-0.3.5.8-src/configure.in~	Sun Dec  2 11:57:09 2001
+++ TeXmacs-0.3.5.8-src/configure.in	Sun Dec  2 11:57:09 2001
@@ -26,16 +26,6 @@
 AC_PROG_CC
 AC_PROG_CXX
 AC_LANG_CPLUSPLUS
-AC_MSG_CHECKING(whether GNU C++ compiler is default)
-case "$GXX" in
-  yes )
-    AC_MSG_RESULT(yes)
-  ;;
-  *)
-    AC_MSG_WARN(no)
-    CXX="g++"
-  ;;
-esac
 GUILE_FLAGS
 CPP_X_HEADERS
 
@@ -252,80 +242,6 @@
 AC_SUBST(CONFIG_WORD_MASK)
 AC_SUBST(CONFIG_MAX_FAST)
 
-#--------------------------------------------------------------------
-# Default paths for installation
-#--------------------------------------------------------------------
-
-if test "$prefix" = ""; then
-  prefix=/usr/local
-fi
-if test "$prefix" = "NONE"; then
-  prefix=/usr/local
-fi
-if test "$exec_prefix" = ""; then
-  exec_prefix=${prefix}
-fi
-if test "$exec_prefix" = "NONE"; then
-  exec_prefix=${prefix}
-fi
-if test "$exec_prefix" = '${prefix}'; then
-  exec_prefix=${prefix}
-fi
-if test "$includedir" = ""; then
-  includedir=${exec_prefix}/include
-fi
-if test "$includedir" = "NONE"; then
-  includedir=${exec_prefix}/include
-fi
-if test "$includedir" = '${exec_prefix}/include'; then
-  includedir=${exec_prefix}/include
-fi
-if test "$libdir" = ""; then
-  libdir=${exec_prefix}/lib
-fi
-if test "$libdir" = "NONE"; then
-  libdir=${exec_prefix}/lib
-fi
-if test "$libdir" = '${exec_prefix}/lib'; then
-  libdir=${exec_prefix}/lib
-fi
-if test "$bindir" = ""; then
-  bindir=${exec_prefix}/bin
-fi
-if test "$bindir" = "NONE"; then
-  bindir=${exec_prefix}/bin
-fi
-if test "$bindir" = '${exec_prefix}/bin'; then
-  bindir=${exec_prefix}/bin
-fi
-if test "$datadir" = ""; then
-  datadir=${prefix}/share
-fi
-if test "$datadir" = "NONE"; then
-  datadir=${prefix}/share
-fi
-if test "$datadir" = '${prefix}/share'; then
-  datadir=${prefix}/share
-fi
-if test "$mandir" = ""; then
-  mandir=${prefix}/man
-fi
-if test "$mandir" = "NONE"; then
-  mandir=${prefix}/man
-fi
-if test "$mandir" = '${prefix}/man'; then
-  mandir=${prefix}/man
-fi
-if test "$libexecdir" = ""; then
-  libexecdir=${exec_prefix}/libexec
-fi
-if test "$libexecdir" = "NONE"; then
-  libexecdir=${exec_prefix}/libexec
-fi
-if test "$libexecdir" = '${exec_prefix}/libexec'; then
-  libexecdir=${exec_prefix}/libexec
-fi
-
 curdir="`pwd`"
 genccpath=${curdir}/gencc/bin
 tmdir=${PACKAGE}-${VERSION}
diff -Nru TeXmacs-0.3.5.8-src/acinclude.m4 TeXmacs-0.3.5.8-src.new/acinclude.m4
--- TeXmacs-0.3.5.8-src/acinclude.m4	Thu Jan  1 01:00:00 1970
+++ TeXmacs-0.3.5.8-src.new/acinclude.m4	Mon Dec  3 19:41:15 2001
@@ -0,0 +1,29 @@
+#-------------------------------------------------------------------
+# Modify the X include files to make them C++-compatible, if needed
+#-------------------------------------------------------------------
+
+AC_DEFUN([CPP_X_HEADERS],[
+AC_MSG_CHECKING(for C++-compatible X header files)
+ac_save_cppflags="$CPPFLAGS"
+CPPFLAGS="$CPPFLAGS $X_CFLAGS"
+AC_TRY_COMPILE([#include <X11/Xlib.h>
+#include <X11/Xutil.h>],,
+echo yes,
+[rm -rf X11
+mkdir X11
+if [ "x$x_includes" != xNONE ]
+then
+  sed 's/^extern \([[a-zA-Z0-9]]*(\)/extern int \1/' \
+    < "$x_includes/X11/Xlib.h" > X11/Xlib.h
+  sed 's/^extern \([[a-zA-Z0-9]]*(\)/extern int \1/' \
+    < "$x_includes/X11/Xutil.h" > X11/Xutil.h
+else
+  sed 's/^extern \([[a-zA-Z0-9]]*(\)/extern int \1/' \
+    < /usr/include/X11/Xlib.h > X11/Xlib.h
+  sed 's/^extern \([[a-zA-Z0-9]]*(\)/extern int \1/' \
+    < /usr/include/X11/Xutil.h > X11/Xutil.h
+fi
+X_CFLAGS="-I.. $X_CFLAGS"
+echo "no; fixing"])
+CPPFLAGS="$ac_save_cppflags"
+])
diff -Nru TeXmacs-0.3.5.8-src/aclocal.m4 TeXmacs-0.3.5.8-src.new/aclocal.m4
--- TeXmacs-0.3.5.8-src/aclocal.m4	Wed Nov 28 09:36:52 2001
+++ TeXmacs-0.3.5.8-src.new/aclocal.m4	Thu Jan  1 01:00:00 1970
@@ -1,54 +0,0 @@
-
-#-------------------------------------------------------------------
-# Modified version of GUILE_FLAGS in guile.m4
-# from the official guile distribution
-#-------------------------------------------------------------------
-
-AC_DEFUN([GUILE_FLAGS],[
-## The GUILE_FLAGS macro.
-  ## First, let's just see if we can find Guile at all.
-  AC_MSG_CHECKING(for Guile)
-  guile-config link > /dev/null || {
-    echo "configure: cannot find guile-config; is Guile installed?" 1>&2
-    exit 1
-  }
-  GUILE_ORIGINAL_CFLAGS="`guile-config compile`"
-  GUILE_CFLAGS="$GUILE_ORIGINAL_CFLAGS"
-  GUILE_VARIANT_CFLAGS="$GUILE_ORIGINAL_CFLAGS $GUILE_ORIGINAL_CFLAGS/guile $GUILE_ORIGINAL_CFLAGS/libguile"
-  GUILE_LDFLAGS="`guile-config link`"
-  GUILE_VARIANT_LDFLAGS="-L`guile-config info libdir` -lguile -lreadline -ltermcap"
-  AC_SUBST(GUILE_CFLAGS)
-  AC_SUBST(GUILE_LDFLAGS)
-  AC_MSG_RESULT(yes)
-])
-
-
-#-------------------------------------------------------------------
-# Modify the X include files to make them C++-compatible, if needed
-#-------------------------------------------------------------------
-
-AC_DEFUN(CPP_X_HEADERS,[
-AC_MSG_CHECKING(for C++-compatible X header files)
-ac_save_cppflags="$CPPFLAGS"
-CPPFLAGS="$CPPFLAGS $X_CFLAGS"
-AC_TRY_COMPILE([#include <X11/Xlib.h>
-#include <X11/Xutil.h>],,
-echo yes,
-[rm -rf X11
-mkdir X11
-if [ "x$x_includes" != xNONE ]
-then
-  sed 's/^extern \([[a-zA-Z0-9]]*(\)/extern int \1/' \
-    < "$x_includes/X11/Xlib.h" > X11/Xlib.h
-  sed 's/^extern \([[a-zA-Z0-9]]*(\)/extern int \1/' \
-    < "$x_includes/X11/Xutil.h" > X11/Xutil.h
-else
-  sed 's/^extern \([[a-zA-Z0-9]]*(\)/extern int \1/' \
-    < /usr/include/X11/Xlib.h > X11/Xlib.h
-  sed 's/^extern \([[a-zA-Z0-9]]*(\)/extern int \1/' \
-    < /usr/include/X11/Xutil.h > X11/Xutil.h
-fi
-X_CFLAGS="-I.. $X_CFLAGS"
-echo "no; fixing"])
-CPPFLAGS="$ac_save_cppflags"
-])
