--- TeXmacs-1.0.0.20-src/configure.in	2002-10-27 01:05:30.000000000 +0200
+++ TeXmacs-1.0.0.20-src.new/configure.in	2002-10-29 19:29:32.000000000 +0100
@@ -12,7 +12,8 @@
 #
 #--------------------------------------------------------------------
 
-AC_INIT(Makefile.in)
+AC_INIT
+AC_CONFIG_SRCDIR([Makefile.in])
 
 PACKAGE="TeXmacs"
 VERSION="1.0.0.20"
@@ -25,7 +26,7 @@
 
 AC_PROG_CC
 AC_PROG_CXX
-AC_LANG_CPLUSPLUS
+AC_LANG([C++])
 AC_MSG_CHECKING(whether GNU C++ compiler is default)
 case "$GXX" in
   yes )
@@ -590,80 +591,6 @@
 AC_SUBST(CONFIG_LIB_PATH)
 AC_SUBST(CONFIG_STRIP)
 
-#--------------------------------------------------------------------
-# Default paths for installation
-#--------------------------------------------------------------------
-
-if test "$prefix" = ""; then
-  prefix=/usr/local
-fi
-if test "$prefix" = "NONE"; then
-  prefix=/usr/local
-fi
-if test "$exec_prefix" = ""; then
-  exec_prefix=${prefix}
-fi
-if test "$exec_prefix" = "NONE"; then
-  exec_prefix=${prefix}
-fi
-if test "$exec_prefix" = '${prefix}'; then
-  exec_prefix=${prefix}
-fi
-if test "$includedir" = ""; then
-  includedir=${exec_prefix}/include
-fi
-if test "$includedir" = "NONE"; then
-  includedir=${exec_prefix}/include
-fi
-if test "$includedir" = '${exec_prefix}/include'; then
-  includedir=${exec_prefix}/include
-fi
-if test "$libdir" = ""; then
-  libdir=${exec_prefix}/lib
-fi
-if test "$libdir" = "NONE"; then
-  libdir=${exec_prefix}/lib
-fi
-if test "$libdir" = '${exec_prefix}/lib'; then
-  libdir=${exec_prefix}/lib
-fi
-if test "$bindir" = ""; then
-  bindir=${exec_prefix}/bin
-fi
-if test "$bindir" = "NONE"; then
-  bindir=${exec_prefix}/bin
-fi
-if test "$bindir" = '${exec_prefix}/bin'; then
-  bindir=${exec_prefix}/bin
-fi
-if test "$datadir" = ""; then
-  datadir=${prefix}/share
-fi
-if test "$datadir" = "NONE"; then
-  datadir=${prefix}/share
-fi
-if test "$datadir" = '${prefix}/share'; then
-  datadir=${prefix}/share
-fi
-if test "$mandir" = ""; then
-  mandir=${prefix}/man
-fi
-if test "$mandir" = "NONE"; then
-  mandir=${prefix}/man
-fi
-if test "$mandir" = '${prefix}/man'; then
-  mandir=${prefix}/man
-fi
-if test "$libexecdir" = ""; then
-  libexecdir=${exec_prefix}/libexec
-fi
-if test "$libexecdir" = "NONE"; then
-  libexecdir=${exec_prefix}/libexec
-fi
-if test "$libexecdir" = '${exec_prefix}/libexec'; then
-  libexecdir=${exec_prefix}/libexec
-fi
-
 curdir="`pwd`"
 tmdir=${PACKAGE}-${VERSION}
 tmsrc=${curdir}/${tmdir}
@@ -678,4 +605,5 @@
 # Done
 #--------------------------------------------------------------------
 
-AC_OUTPUT(Makefile:Makefile.in src/common.makefile:src/common.makefile.in src/Basic/System/tm_configure.hh:src/Basic/System/tm_configure.in misc/Makefile:misc/Makefile.in misc/doxygen/Doxyfile:misc/doxygen/Doxyfile.in misc/scripts/fig2ps:misc/scripts/fig2ps.in misc/scripts/texmacs:misc/scripts/texmacs.in)
+AC_CONFIG_FILES([Makefile:Makefile.in src/common.makefile:src/common.makefile.in src/Basic/System/tm_configure.hh:src/Basic/System/tm_configure.in misc/Makefile:misc/Makefile.in misc/doxygen/Doxyfile:misc/doxygen/Doxyfile.in misc/scripts/fig2ps:misc/scripts/fig2ps.in misc/scripts/texmacs:misc/scripts/texmacs.in])
+AC_OUTPUT
