diff -urN TeXmacs-0.3.5.11-src.org/gencc/src/gg_system.in TeXmacs-0.3.5.11-src/gencc/src/gg_system.in
--- TeXmacs-0.3.5.11-src.org/gencc/src/gg_system.in	Sat Jan  5 11:50:15 2002
+++ TeXmacs-0.3.5.11-src/gencc/src/gg_system.in	Sat Jan  5 11:50:54 2002
@@ -98,7 +98,7 @@
 make_dynamic_binary (string in, string out, string options, string lib_path) {
 #ifdef __alpha
   return
-    "$(LDB) $(LDBFLAGS) -lstdc++ -lg++ $(LDBSHARED) -L" * lib_path * " " *
+    "$(LDB) $(LDBFLAGS) -lstdc++ $(LDBSHARED) -L" * lib_path * " " *
     options * (N(options)==0? "": " ") *
     // "$(LDBPATH)" * lib_path * " " *
     in * " -o " * out;
